numba/np/random/old_distributions.py

```diff
@@ -673,6 +673,8 @@ def random_binomial_btpe(bitgen, n, p):
                       / w2) / w / 66320.)):
                 case = 10
                 continue
+            case = 60
+            continue
```

Comment:
Confirmed this change corresponds to the fallthrough in https://github.com/numpy/numpy/blob/aab644b3ceb8742e2091c8cc51ff932e14b0627e/numpy/random/src/distributions/distributions.c#L752-L754